src/assets/badges/svgXMLs/dryers.ts

```diff
@@ -11,4 +11,175 @@ const dryersXML = `<svg width="76" height="70" viewBox="0 0 76 70" fill="none" x
 </svg>
 `;
 
-export default dryersXML;
+const grayDryersXML = `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="76"
```

Comment:
why are the width and height for all of these different?

Reply:
Yes, as the dimensions are obtained from the figma. I don't think the tool I used to grayscale these (Inkscape) modifies height and width of the SVG, so I think this just came from figma itself.